src/main/java/com/example/loadtest/domain/order/service/OrderServiceImpl.java

```diff
@@ -0,0 +1,88 @@
+package com.example.loadtest.domain.order.service;
+
+import com.example.loadtest.domain.order.dto.OrderItemDTO;
+import com.example.loadtest.domain.order.dto.OrderRequestDTO;
+import com.example.loadtest.domain.order.dto.OrderResponseDTO;
+import com.example.loadtest.domain.order.entity.*;
+import com.example.loadtest.domain.order.repository.OrderRepository;
+import com.example.loadtest.domain.store.entity.Menu;
+import com.example.loadtest.domain.store.entity.Store;
+import com.example.loadtest.domain.store.repository.MenuRepository;
+import com.example.loadtest.domain.store.repository.StoreRepository;
+import com.example.loadtest.domain.user.entity.User;
+import com.example.loadtest.domain.user.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class OrderServiceImpl implements OrderService {
+
+    private final OrderRepository orderRepository;
+    private final UserRepository userRepository;
+    private final StoreRepository storeRepository;
+    private final MenuRepository menuRepository;
+
+    @Override
+    @Transactional
+    public Long createOrder(OrderRequestDTO dto) {
```

Comment:
Wrapping type 대신 primitive 타입으로 수정하면 좋을 것 같네요.
```suggestion
    public long createOrder(OrderRequestDTO dto) {
```